### DISCUSSION TITLE
Offensive Cybersecurity

### DISCUSSION THREAD

**Discussion by dkeseloff** [NillionNetwork/discussions#152] [Ideas]
I have an associate who is an expert white hat hacker, with govt clearance and has taken over the whole network of govt agencies (with their written permission, of course!)

Do we need end-to-end penetration testing?

Let me know who I need to talk to on this.

**Comment by wwwehr** [comment]
Thanks for the reach out. We're covered wrt audits at the moment. Best regards!
